pkgs/applications/editors/l3afpad/default.nix

```
@@ -0,0 +1,25 @@
+{ lib, stdenv, fetchurl, intltool, pkg-config, gtk3, fetchFromGitHub
+, autoreconfHook, wrapGAppsHook }:
+
+stdenv.mkDerivation rec {
+  version = "unstable-2022-02-17";
```

From what I am seeing here: https://github.com/stevenhoneyman/l3afpad/commit/16f22222116b78b7f6a6fd83289937cdaabed624

```suggestion
  version = "unstable-2022-02-14";
```